### DISCUSSION TITLE
http://localhost:3000/players/profile?id=52872658083619

### DISCUSSION THREAD

**Discussion by giscus** [PanosPasch/efootballdb-giscus#1] [General]
# http://localhost:3000/players/profile?id=52872658083619

eFootball DB M. Salah Player Page - M. Salah player from eFootball 2022 by KONAMI

http://localhost:3000/players/profile?id=52872658083619

**Comment by PanosPasch** [comment]
Testing comment for mohammed salah

**Reply by PanosPasch** [reply]
testing reply for your comment
